docs/radio.rst

```diff
@@ -0,0 +1,129 @@
+* Messages are broadcast and received on a preselected channel (there are 100 available channels to choose from).
+* Broadcasts are at a certain level of power - more power means more range.
+* Messages are filtered by address (like a house number) and group (like a named recipient at the specified address).
+* The rate of throughput can be one of three pre-determined settings.
+* Send and receieve bytes to send arbitrary data.
+* As a convenience for children, it's easy to send and receive messages as strings.
+* The default configuration is both sensible and compatible with other platforms that target the BBC micro:bit.
+
+To access this module you need to::
+
+    import radio
+
+We assume you have done this for the examples below.
+
+Constants
+=========
+
+.. py:attribute:: RATE_250KBIT
+
+    Constant used to indicate a throughput of 256 Kbit a second.
+
+.. py:attribute:: RATE_1MBIT
+
+    Constant used to indicate a throughput of 1 MBit a second.
+
+.. py:attribute:: RATE_2MBIT
+
+    Constant used to indicate a throughput of 2 MBit a second.
+
+
+Functions
+=========
+
+.. py:function:: on()
+
+    Turns the radio on. This needs to be explicitly called since the radio
+    draws power and takes up memory that you may otherwise need.
+
+.. py:function:: off()
+
+    Turns off the radio, thus saving power and memory.
+
+.. py:function:: config(length=32, queue=3, channel=7, power=0, address=0x75626974, group=0, data_rate=radio.RATE_1MBIT)
+
+    Configures the various settings relating to the radio. The specified
+    default values are sensible.
+
+    The ``length`` defines the maximum length, in bytes, of a message sent via
+    the radio. It can be up to 251 bytes long (254 - 3 bytes for S0, LENGTH
+    and S1 preamble).
+
+    The ``queue`` specifies the number of messages that can be stored on the
+    incoming message queue. If there are no spaces left on the queue for
+    incoming messages, then the incoming message is dropped.
+
+    The ``channel`` can be an integer value from 0 to 100 (inclusive) that
+    defines an arbitrary "channel" to which the radio is tuned. Messages will
+    be sent via this channel and only messages received via this channel will
+    be put onto the incoming message queue. Each step is 1MHz wide, based at
+    2400MHz.
+
+    The ``power`` is an integer value from 0 to 7 (inclusive) to indicate the
+    strength of signal used when broadcasting a message. The higher the value
+    the stronger the signal, but the more power is consumed by the device. The
+    numbering translates to positions in the following list of dBm (decibel
+    milliwatt) values: -30, -20, -16, -12, -8, -4, 0, 4.
+
+    The ``address`` is an arbitrary name, expressed as a 32-bit address, that's
+    used to filter incoming packets at the hardware level, keeping only those
+    that match the address you set. The default used by other micro:bit related
+    platforms is ``b'uBit\x00'``. It must be exactly 5 bytes in length.
```

Comment:
Change `b'uBit'` to 0x75626974.  And remove sentence "It must be exactly 5 bytes in length.".